cmd/app/use.go

```diff
@@ -0,0 +1,108 @@
+package app
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/stormkit-io/stormkit-cli/api"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+	"github.com/spf13/cobra"
+)
+
+// useCmd represents the use command
+var useCmd = &cobra.Command{
+	Use:   "use",
+	Short: "Select an application to work on",
+	Long: `Select an application where to work on, via his repo name, id or index.
+Via his repo name:
+
+$ stormkit-cli app use <repo_name>
+
+Via his id:
+
+$ stormkit-cli app use --app-id <app_id>
+
+Via his index:
+
+$ stormkit-cli app ls -n
+0 github/myuser/my-project
+$ stormkit-cli app use 0
+
+This command is useful when you are not in the directory of your repository.
+It keeps his memory in the config file, at: app.engine.app_id.
+Be carefull this wouldn't work if you are in the direcotry of another 
+stormkit project.`,
+	RunE: runAppUse,
+}
+
+func init() {
+	appCmd.AddCommand(useCmd)
+
+	useCmd.Flags().StringP("app-id", "a", "", "ID of the app to use")
```

Comment:
**suggestion** 

WDYT about rephrasing

```
"ID of the app to use"
```

to

```
The application ID to use
```

Reply:
Wait review on [this](https://github.com/stormkit-io/stormkit-cli/pull/1#discussion_r516257357)

Reply:
the `app-id` flag not exists anymore.